src/get-started/test-service.md

```diff
@@ -379,5 +391,5 @@ logic and HTTP API.
 [integration-testing]: http://doc.crates.io/manifest.html#integration-tests
```

Comment:
Nit: Could you please update the link to point to `https://doc.rust-lang.org/cargo/reference/manifest.html#integration-tests`?